Assignment2/marialucas96-MariaLucas.turtle.rdf

```diff
@@ -0,0 +1,9 @@
+	
+@base <http://www.ontogrid.net/StickyNote#>.
+@prefix xmls: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 :includes :Sensor029; :includes :Computer101.
+:Sensor029 :hasMeasurement :Measurement8401.
+:Measurement8401 :hasTemperature 29^^xmls:integer; :atTime 2010-06-12T12:00:12^^xmls:dateTime.
```

Comment:
Still lots of issues with the syntax of this Turlte file. I cannot understand why you are still making these mistakes, which already appeared in other student's assignment.